packages/cli/package.json

```diff
@@ -7,7 +7,7 @@
     "postinstall": "mkdir -p input; mkdir -p output",
     "precommit": "lint-staged",
     "start": "dotenv -- npm run start:transform",
-    "start:transform": "ts-node index.ts transform --from s3 --to stdout --project $TRANSFORM_PROJECT --debug -v s3endpoint=$S3_ENDPOINT -v s3bucket=$S3_BUCKET -v graph-store-endpoint=$GRAPH_STORE_ENDPOINT -v graph-store-user=$GRAPH_STORE_USER -v graph-store-password=$GRAPH_STORE_PASSWORD -v graph=$TRANSFORM_GRAPH",
+    "start:transform": "ts-node index.ts transform --from s3 --to stdout --project $TRANSFORM_PROJECT --debug --enable-buffer-monitor -v s3endpoint=$S3_ENDPOINT -v s3bucket=$S3_BUCKET -v graph-store-endpoint=$GRAPH_STORE_ENDPOINT -v graph-store-user=$GRAPH_STORE_USER -v graph-store-password=$GRAPH_STORE_PASSWORD -v graph=$TRANSFORM_GRAPH",
```

Comment:
Do we really want to enable the new option by default? I thought it was for debugging purpose only

Reply:
the npm script is just for development time anyway

Reply:
And it won't break the debug output?

Anyway, that's your call. We can change it later if it's annoying.

Reply:
Nothing really "breaks". I also wanted to keep it like this for now so that when Adrian tries to run from sources, which he does, then he will see for himself how those options behave together ;)